server/lib/donations.js

```diff
@@ -32,7 +31,7 @@ export const processDonation = (Sequelize, donation) => {
                   groupId: group.id,
                   groupName: group.name,
                   paymentMethodId: paymentMethod.id,
-                  description: `OpenCollective: ${group.slug}`
+                  description: `https://opencollective.com/${group.slug}`
```

Comment:
It doesn't cost us anything to add and one day we may very well be thankful that we did that. I could see a host like women who code make use of that info as they parse their stripe data (they already asked us to add the slug data there)